### PR TITLE
Add deprecateCapability

### PR DESCRIPTION
**Pull request by DeividasK**

### CODE REVIEW

File: contracts/src/v0.8/keystone/CapabilityRegistry.sol [hunk 1]
```diff
@@ -74,6 +74,16 @@ contract CapabilityRegistry is OwnerIsCreator, TypeAndVersionInterface {
...
 
+  /// @notice This error is thrown when a capability with the provided ID is
+  /// not found.
+  /// @param capabilityId The ID used for the lookup.
+  error CapabilityDoesNotExist(bytes32 capabilityId);
+
+  /// @notice This error is thrown when trying to deprecate a capability that
+  /// is already deprecated.
+  /// @param capabilityId The ID of the capability that is already deprecated.
+  error CapabilityIsDeprecated(bytes32 capabilityId);
```
Comment (bolekk) [inline contracts/src/v0.8/keystone/CapabilityRegistry.sol:85]: This name is very similar to the event name - maybe rename to "CapabilityAlreadyDeprecated"? Or even consider not throwing an error at all?

File: contracts/src/v0.8/keystone/test/CapabilityRegistry_DeprecateCapability.t.sol [hunk 2]
```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: MIT
```
Comment (bolekk) [inline contracts/src/v0.8/keystone/test/CapabilityRegistry_DeprecateCapability.t.sol:1]: nit: file name correct?

**Review by bolekk — APPROVED** [review]

File: contracts/src/v0.8/keystone/CapabilityRegistry.sol (hunk 1 above)
Comment (DeividasK) [reply contracts/src/v0.8/keystone/CapabilityRegistry.sol]: Good shout, renamed 👍

File: contracts/src/v0.8/keystone/test/CapabilityRegistry_DeprecateCapability.t.sol (hunk 2 above)
Comment (DeividasK) [reply contracts/src/v0.8/keystone/test/CapabilityRegistry_DeprecateCapability.t.sol:1]: This tests `deprecateCapability`, so I'm following the convention of naming files with the function that is being tested.

**Review by archseer — APPROVED** [review]

**Pull request merged**